Makefile

```diff
@@ -163,3 +163,7 @@ clean:
 .PHONY: changelog
 changelog:
 	git log --color=always --first-parent --pretty='format:%s|%Cgreen%d%Creset' | column -ts '|' | less "$(lessopts)"
+
+.PHONY: primevue
+primevue:
+	cd primevue-gen && npm i && npm run build
```

Comment:
This can be improved so the build happens only when necessary. Look at the code below:

```
.PHONY: primevue
# By building primevue the following objects will be generated:
# - primevue-gen/node_modules
# - promgen/static/primevue/main.css
# - promgen/static/primevue/main.js
# Therefore, they should be added as dependencies.
#
# Also, let's create rules for each one of those dependencies, so they will be built
# when they are missing. This primevue rule does nothing but listing dependencies.
primevue: primevue-gen/node_modules promgen/static/primevue/main.css promgen/static/primevue/main.js

# The node_modules object depends on the package.json, so let's add it to the
# dependencies. That will make this rule to be executed if the package.json changes.
# The only thing needed for building the node_modules object is "npm i".
primevue-gen/node_modules: primevue-gen/package.json
       cd primevue-gen && npm i

# The main.css and main.js objects depend on the build.js and src/main.js files, so
# let's add them to the dependencies. If any of them changes, this rules will be
# executed.
# The only thing needed to build those objects is "npm run build".
promgen/static/primevue/main.css: primevue-gen/build.js primevue-gen/src/main.js
promgen/static/primevue/main.js: primevue-gen/build.js primevue-gen/src/main.js
       cd primevue-gen && npm run build
```

Comments are just for illustration purposes, we don't need to add them.

Reply:
addressed this comments in [b821c93](https://github.com/line/promgen/pull/456/commits/b821c93d281f6f6ad4cd9fa4c3a0e30f2a793043)